lib/matplotlib/offsetbox.py

```diff
@@ -81,6 +117,9 @@ def _get_packed_offsets(wd_list, total, sep, mode="fixed"):
     elif mode == "equal":
         maxh = max(w_list)
         if total is None:
+            if sep is None:
+                raise ValueError("total and sep cannot both be None when "
```

Comment:
This raised a TypeError before in the line below (int + None).

Since the function is private, we can do the change without announcement.